tx/owner.go

```diff
@@ -3,14 +3,14 @@ package tx
 import (
 	"context"
 	"crypto/ecdsa"
-	stderr "errors"
 	"fmt"
 	"math/big"
 	"time"
 
 	ethereum "github.com/ethereum/go-ethereum"
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/core/types"
+	stderr "github.com/pkg/errors"
```

Comment:
Can we use `errors` here instead of `stderr` to be consistent across the repo?

Reply:
unfortunately it clashes with the `errors` package in the gateway itself `github.com/oasislabs/oasis-gateway/errors`